pyriemann_qiskit/classification.py

```diff
@@ -0,0 +1,381 @@
+"""Module for classification function."""
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.model_selection import train_test_split
+
+from qiskit import BasicAer, IBMQ
+from qiskit.circuit.library import ZZFeatureMap, TwoLocal
+from qiskit.aqua import QuantumInstance, aqua_globals
+from qiskit.aqua.quantum_instance import logger
+from qiskit.aqua.algorithms import QSVM, SklearnSVM, VQC
+from qiskit.aqua.utils import get_feature_dimension
+from qiskit.providers.ibmq import least_busy
+from qiskit.aqua.components.optimizers import SPSA
+from datetime import datetime
+import logging
+logger.level = logging.INFO
+
+
+class QuanticClassifierBase(BaseEstimator, ClassifierMixin):
+
+    """Quantum classification.
+
+    This class implements a SKLearn wrapper around Qiskit library.
+    It provides a mean to run classification tasks on a local and
+    simulated quantum computer or a remote and real quantum computer.
+    Difference between simulated and real quantum computer will be that:
+
+    * There is no noise on a simulated quantum computer (so results are better)
+    * Real quantum computer are quicker than simulator
+    * Real quantum computer tasks are assigned to a queue
+      before being executed on a back-end
+
+    WARNING: At the moment this implementation only supports binary
+    classification (eg. Target vs Non-Target experiment)
+
+    Parameters
+    ----------
+    target : int
+        Label of the target symbol
```

Comment:
Code has been deeply improve! Well done!

But I think we can go further, because current description and code have been written for the specific BCI use-case "target versus non-target classification".

Can you write the description and the code for any binary classification, without any reference to target and non-target classes?

For example:
- parameter `target` should be renamed `labels`, being a 2-tuple containing the label of each class;
- variables `Xta` and `Xnt` should be renamed into `X_class0` and `X_class1`,
- etc.

This refactoring will probably help to move towards a multi-class classifier.

Reply:
Sure, you are right. 
It makes me think also to open an issue for the multi-class classifier here: 
https://github.com/pyRiemann/pyRiemann-qiskit/issues/6

Reply:
Ok for the multiclass extension in another PR.
But, in this PR, can you at least rename variables, to provide generic binary classifiers, without any reference to ERP classification?

Reply:
Yes, my message was a bit confusing. I confirm I am doing the renaming of the variable here. 
For the multiclass extension, I will need to investigate it a bit, as I never did it with QS. This is why I created another issue.